### ISSUE TITLE
Q: some parts can be use in Osmose-QA ?

### ISSUE THREAD

**Issue by frodrigo** [pierzen/OQ_Analysis#1]
Osmose-QA also use an Osmosis database. Do you think some parts can be reused or shared in osmose QA ?

**Comment by pierzen** [comment]
The Topological Analysis section compares buildings and other features such as riverbanks, highways, landuse and detect  building self-crossing, buildings superpositions or buildings superposed to other features. I think that this information is already integrated into Osmose-QA.

The other section, Warnings about Buildings Geometry evaluation (ie Regular vs Irregular) is thought to monitor mapping for a territory or quality of Imports. While a 5% ratio of Irregular buildings in a territory sounds raisonnable, we have noticed areas with over 60% of buildings with irregular forms. Looking closer, it did show very imprecise mapping as reported on the [OpenDatalabRDC Blog](https://opendatalabrdc.github.io/Blog).  In december 2018, noticing poor quality mapping for Butembo Ebola response in DR Congo, we were able to asssess rapidly the situation using these scripts. It did also provide us the list of Duplicates or superposed polygons for rapid correction.

We also had discussions recently on Talk-ca about Building imports with high ratio of irregular forms that needed some review.

I think that adding individual building Geometry evaluation into Osmose-QA would create too many false positives about irregularities.
